seaborn/categorical.py

```diff
@@ -1840,24 +1840,51 @@ def _width_functions(self, width_func):
 
     def _lvplot(self, box_data, positions,
                 color=[255. / 256., 185. / 256., 0.],
-                widths=1, ax=None, **kws):
+                widths=1, ax=None, box_kws=None,
+                flier_kws=None,
+                line_kws=None):
+
+        # -- Default keyword dicts - based on
+        # distributions.plot_univariate_histogram
+        box_kws, flier_kws, line_kws = map(
+            lambda d: {} if d is None else d.copy(),
+            (box_kws, flier_kws, line_kws))
+
+        # Set the default kwargs for the lines denoting medians
+        line_default_kws = dict(
+            color=".15",
+            alpha=0.45,
+            solid_capstyle="butt",
+            linewidth=self.linewidth,
+        )
+        # Set the default kwargs for the outliers scatterplot
+        flier_default_kws = dict(marker='d', color=self.gray)
+
+        # Update each of the dicts containing box/line/fliers kwargs with the
+        # default values (if not defined by the user)
+        default_dicts = [line_default_kws, flier_default_kws]
+        user_input_dicts = [line_kws, flier_kws]
+        for default_dict, user_dict in zip(default_dicts, user_input_dicts):
+            for k, v in default_dict.items():
+                user_dict.setdefault(k, v)
 
         vert = self.orient == "v"
         x = positions[0]
         box_data = np.asarray(box_data)
 
         # If we only have one data point, plot a line
         if len(box_data) == 1:
-            kws.update({
-                'color': self.gray, 'linestyle': '-', 'linewidth': self.linewidth
+            line_kws.update({
```

Comment:
I think this should be sourcing kwargs from `box_kws`, after you resolve the default/custom options:

- `edgecolor` -> `color`
- `linewidth` -> `linewidth`
- `linestyle` -> `linestyle`

Reply:
(I am not sure about how to handle the corner case where the box linewidth is 0, which would be valid for boxes but make this artist invisible. Perhaps `linewidth=max(box_kws["linewidth"], line_kws["linewidth"])`?

Reply:
The solution seems good to me. 

`linestyle` is not defined by default on box_kws, hence the added `box.kws` call.